### PR TITLE
[Doc Improvement][Changes in TeamsFx command line interface][3411323]

### PR DESCRIPTION
**Pull request by v-vanisreeth**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b0f4ea3eb4b77d38501171bc9b6494c5f1cbdfc1/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 92 | 46 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5b61e705-943a-419b-9295-9fbe71f31d47) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-CLI.md [hunk 1]
```diff
@@ -569,7 +569,7 @@ Preview the current application from local or remote.
 |`--browser`| No | The browser to open Teams web client. The options are `chrome`, `edge` and `default` such as system default browser and the value is `default`. |
 |`--browser-arg`| No | Argument to pass to the browser, requires --browser, can be used multiple times, for example, --browser-args="--guest" |
 |`--sharepoint-site`| No | SharePoint site URL, such as `{your-tenant-name}.sharepoint.com` for SPFx project remote preview. |
-|`--m365-host`| Preview the application in Teams, Outlook or Microsoft 365 (formally known as Office app). Options are `teams`, `outlook` and `office`. The default value is `teams`. |
+|`--m365-host`| No |Preview the application in Teams, Outlook or Microsoft 365 (formally known as Office app). Options are `teams`, `outlook` and `office`. The default value is `teams`. |
```
Comment (Wajeed-msft) [inline msteams-platform/toolkit/TeamsFx-CLI.md:572]: I believe header value for column should to `Required` instead of `Requirement` . Please check.

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7dced3f60639bedf2abce416d263fb628a3932c/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 92 | 46 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/434f9373-45e8-4b11-96ec-2aa4199d36f0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-CLI.md (hunk 1 above)
Comment (v-vanisreeth) [reply msteams-platform/toolkit/TeamsFx-CLI.md:572]: I've changed per the above feedback.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d429949ab9cc4bb2d4cae767b15d9a79d6a3ba29/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 92 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd3a66f5-f686-4441-9067-cf92f91e8112) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5f93d512f76d7679be93fbac3215ea6d7e9f4e95/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 92 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a75d62dc-e2e5-4abf-9584-548ab40b9068) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/TeamsFx-CLI.md](https://github.com/MicrosoftDocs/msteams-docs/blob/81c447b45f6433a606a1b976d4b2f13a93161cd1/msteams-platform%2Ftoolkit%2FTeamsFx-CLI.md) | 92 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96004738-19ec-41b8-b2be-1f72750123d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**